benchmarks/frameworks/torch/mpc/scripts/benchmark_tanh.py

```diff
@@ -0,0 +1,104 @@
+"""
+This script is to benchmark the tanh function methods.
+For more info see: https://github.com/OpenMined/PySyft/issues/3999
+"""
+
+from workers_initialization import workers, hook
+from benchmark_sample_data import benchmark_data_tanh
+import torch
+import timeit
+import matplotlib.pyplot as plt
+
+
+def benchmark_tanh(method, prec_frac, workers):
+    """
+    This function approximates the tanh function using a given method.
+
+    Args:
+        method (str): the name of the method for approximation
+        prec_frac (int): precision value
+        workers (dict): workers used for sharing data
+
+    Returns:
+        diff (int): the difference between the syft and torch approximated value
+
+    """
+    alice, bob, james = workers["alice"], workers["bob"], workers["james"]
+
+    t = torch.tensor([1.23212])
+    t_sh = t.fix_precision(precision_fractional=prec_frac).share(alice, bob, crypto_provider=james)
+    r_sh = t_sh.tanh(method=method)
+    r = r_sh.get().float_prec()
+    t = t.tanh()
+    # Calculation of the difference between FPT and normal sigmoid (error)
+    diff = (r - t).abs().max()
+    return diff.item()
+
+
+def tanh_approximation_plot(benchmark_data_tanh):
+    """
+    This function plots the graph for various sigmoidal approximation benchmarks namely
+    'chebyshev', 'maclaurin', 'exp'.
+
+    Args:
+        benchmark_data_sigmoid (list): the sample data to approximate
+
+    Returns:
+        sigmoid_function_approximations_benchmark (png): plotted graph in graph directory
```

Comment:
Incorrect doc-strings :stuck_out_tongue:

Reply:
Ah! You're right! Fixed this now.